adafruit_tsl2591.py

```diff
@@ -39,7 +39,7 @@
 
 #pylint: disable=bad-whitespace
 # Internal constants:
-_TSL2591_ADDR                = const(0x29)
+_TSL2591_ADDR                = 0x29
```

Comment:
From an email thread with @tdicola:

```
why the change from a const for the I2C address to non-const value in all these pulls? 
That's a small (and admittedly low risk) change to the function of the code, not just the docs.
```

Good point! This improves how the default value is rendered by sphinx. With the `const()` it ends up as a MockObject to sphinx like [this](https://circuitpython.readthedocs.io/projects/bme280/en/latest/api.html#adafruit_bme280.Adafruit_BME280_I2C). Changing it to a regular number means it shows the decimal value. We could look into mocking `const `out properly instead.

Reply:
Yeah the const choice is an explicit decision to reduce memory usage though.  I would rather we err on the side of efficiency.  Does sphinx have a way to manually override the generated doc in this case?  That would be the best option if we don't like the mock object there.

Reply:
Now that I think about it, a better option is probably having a pip dependency that simply has const as a lambda. I feel like this exists somewhere.